### ISSUE TITLE
[MAS1.4.3] [Visual Requirements- Add language understanding] Luminosity Ratio is less than 4.5:1 for “add a LUIS app manually “ link present in the LUIS application dialog

### ISSUE THREAD

**Issue by ShikhaMishra11** [microsoft/BotFramework-Emulator#1965]
**Actual Result:**
Luminosity Ratio of  “add a LUIS app manually “ link present in the LUIS application dialog is less than 4.5:1.

**Expected Result:**
Luminosity Ratio should be greater than or equal to 4.5:1 for “add a LUIS app manually “ link.

**Note:** 
Same issue found with the links present at various dialog's throughout the application.

**Observation:**
Links available at "Welcome" and "Settings" screen have sufficient color contrast.

**Repro Steps:**
1. Open the Bot Framework V4 Emulator.
2. Welcome tab screen gets opened.
3. Navigate to any already created bot and select any bot..
4. "Live Chat" screen will appear.
5. Navigate to "+" Services symbol and select it 
6. Navigate to the Add Language understanding(LUIS) and select it 
7. Dialog gets opened 
8. Open 'Color Contrast Analyzer' application.
9. Verify whether Luminosity Ratio is less than 4.5:1 for “add a LUIS app manually “or not 

**User Impact:**
Users with low vision or color blindness needs that difference between contrast of link is always high.

**Test Environment:**
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.5.2-linux

**Impacted MAS Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)

**Attachment:**
![Luminosity Ratio is less than 4 51 for “add a LUIS app manually “ link](https://user-images.githubusercontent.com/55279608/68110586-4c3c6a00-ff13-11e9-9569-f8b4a3920618.png)
![CCA of links available at Dispatch model](https://user-images.githubusercontent.com/55279608/68110606-53637800-ff13-11e9-97ce-4ceb396e2e90.png)
![CCA of links available at Endpoint](https://user-images.githubusercontent.com/55279608/68110608-53fc0e80-ff13-11e9-90cf-5e373150ceba.png)
![CCA of link available at QnA Maker](https://user-images.githubusercontent.com/55279608/68110610-53fc0e80-ff13-11e9-9cda-efba0e8182d4.png)

**Comment by ShikhaMishra11** [comment]
We have checked in the latest nightly, the issue is fixed now. So, we are closing the bug.

Test Environment:
OS: Ubuntu 18.04.3 LTS
Application: BotFramework-Emulator-4.6.0-94878
